docs/config.rst

```diff
@@ -318,6 +319,17 @@ Features
 
    See also: :attr:`LP_MARK_BZR`.
 
+.. attribute:: LP_ENABLE_CMAKE
+    :type: bool
```

Comment:
Indentation in the rest of this file is 3 spaces (don't get me started on why, Sphinx started it that way and I didn't think to change it until too late).

Reply:
It's a kind of super-power to be able to spot such discrepencies :open_mouth: